### ISSUE TITLE
Error output from run command does not populate in quickfix window

### ISSUE THREAD

**Issue by hk0i** [kballard/vim-swift#1]
After I run with cmd-r and get some compiler error output I am unable to open it in quickfix via :cw or :copen

**Comment by lilyball** [comment]
Is that something that's actually expected? I thought that was supposed to be populated from `:make`.

**Comment by hk0i** [comment]
I'm assuming the cmd-r should work similar to Xcode's in that it will first build and then run the application, Assuming there are build errors the application won't run.

These errors are already output by cmd-r but it would be useful to be able to get at them from quickfix.

My flow is now, for example:

cmd-r
-fail
cmd-b (runs :make)
-fails again
:cw

instead of having a single press of cmd-r.

I've altered my flow to do the following:
cmd-b
-if build passes
-- cmd-r
-if build fails
-:cw
then when build passes again, cmd-r

I should still be able to :make if I don't want to run, but I would assume that run would do both build and run (which it seems to do already)

**Comment by lilyball** [comment]
It does build and run, but I hadn't even considered the idea that it should populate quickfix. FWIW, I have Syntastic installed, so I use it to highlight my build errors. I rarely even look at the quickfix window.

I'll see what I can do about populating quickfix.

**Comment by lilyball** [comment]
Also note that ⌘R only builds and runs the current file, whereas `:make` can be configured to build a larger project.

**Comment by hk0i** [comment]
Yeah I've given some thought to that too. I'm also using YouCompleteMe right now, which doesn't seem to handle swift at all... is your syntastic configured for swift? I'm pretty sure I can't use them side by side, though... Another thought I had was to make it build the current file on every save

**Comment by lilyball** [comment]
Syntastic isn't a completion plugin, it's a plugin that automatically shows build errors (by attempting to compile every time the file is saved). My vim-swift plugin includes the settings for Syntastic so if you have Syntastic installed it should automatically start working for Swift.

**Comment by hk0i** [comment]
I was under the impression that YCM also obsoleted Syntastic, but I guess it supports only clang ... in any case now that I'm using syntastic the file seems to run every time after i save the file now (the application is single-file so my gui is launched every time after saving) I'll take another look into it a bit later and see if I can figure out why. In any case, the error reporting is nice :)

**Comment by lilyball** [comment]
After saving? That doesn't make sense. Syntastic only compiles the file, it doesn't run it.
